.github/workflows/static.yaml

```diff
@@ -0,0 +1,39 @@
+name: Static
```

Comment:
Then I would name this here CI, because static is a bit misleading, even s prefix in GuthubActions below the PR